engines/sci/graphics/font.cpp

```diff
@@ -87,13 +91,16 @@ void GfxFontFromResource::draw(uint16 chr, int16 top, int16 left, byte color, bo
 	int y = 0;
 	int16 greyedTop = top;
 
-	byte *pIn = getCharData(chr);
+	SciSpan<const byte> charData = getCharData(chr);
+	if (!charData) {
+		error("Character '%c' does not exist in font %d", chr, _resourceId);
+	}
```

Comment:
This is not like it was before.
getCharData defaulted to the first char (which is afaik normally a space) in case an unknown character was asked for. Now it bombs out.

SCI16 behavior worked also exactly like that (using first character for unknown ones). It wasn't some coding error on our end, but intentionally.